src/extensions/default/UploadFiles/styles.css

```diff
@@ -0,0 +1,92 @@
+.drag-files-area {
```

Comment:
I can, but it means that it's basically impossible to edit/maintain.  I'm not sure we save so much by minifying that it's worth losing the ability to maintain it later.